stock_indicators/indicators/adl.py

```diff
@@ -4,7 +4,7 @@
 from stock_indicators._cstypes import List as CsList
 from stock_indicators.indicators.common.helpers import ToQuotesMixin
 from stock_indicators.indicators.common.results import IndicatorResults, ResultBase
-from stock_indicators.indicators.common.quote import Quote
+from stock_indicators.indicators.common.quotes import Quote
```

Comment:
quote seemed more aligned here?  quotes implies a collection.

Reply:
Umm.. You're right. I will close this PR. 😅